### DISCUSSION TITLE
Losing transparent background on current file when using nvim-tree.

### DISCUSSION THREAD

**Discussion by fnacarellidev** [nvim-tree/nvim-tree.lua#1762] [Q&A]
When I open nvim-tree I lose the transparency background on the current file, I need to reopen it, also I'd like to make my nvim-tree transparent too, any ideas on how to do it?
[Here's what's going on](https://imgur.com/a/hEdLKPt)

**Comment by alex-courtis** [comment]
Similar question here: #1758

Let's continue the discussion there. Same questions apply.
